src/__tests__/Cloud-test.js

```diff
@@ -301,7 +301,9 @@ describe('CloudController', () => {
     expect(data).toEqual({
       limit: 1,
       where: {
-        objectId: 'jobId1234',
```

Comment:
Should all these tests be modified? That means the original syntax would not be tested anymore?

Reply:
Could that be a breaking change for existing deployments?